specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json

```diff
@@ -0,0 +1,215 @@
+                    "ConsoleDisabled"
+                ],
+                "operationId": "ListConsole_Disabled",
+                "description": "Gets if Serial Console is disabled for a subscription.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "default",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "Default string modeled as parameter for URL to work correctly.",
+                        "enum": [
+                          "default"
+                        ]
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK - Returns a JSON object"
+                    }
+                },
+                "x-ms-examples": {
+                    "Get whether or not Serial Console is disabled": {
+                    "$ref": "./examples/GetConsoleDisabledExample.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.SerialConsole/providers/consoleServices/{default}/enableConsole": {
+            "post": {
+                "tags": [
+                    "EnableConsole"
+                ],
+                "operationId": "Console_enableconsole",
+                "description": "Enables Serial Console for a VM",
+                "parameters": [
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ConsoleProperties"
+                        },
+                        "description": "Parameters for Console enablement"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "default",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "Default string modeled as parameter for URL to work correctly.",
+                        "enum": [
+                          "default"
+                        ]
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK - Serial Console disabled"
+                    },
+                    "500": {
```

Comment:
error responses like 5xx should not be modeled separately in the response section. There should be  a "default" response to capture all error cases. Please refer to Batch RP swagger for sample